acme/acme/challenges_test.py

```diff
@@ -264,7 +264,9 @@ def test_gen_verify_cert_gen_key(self):
 
     def test_verify_bad_cert(self):
         self.assertFalse(self.response.verify_cert(
-            test_util.load_cert('cert.pem')))
+            OpenSSL.crypto.load_certificate(
```

Comment:
You've uncovered an interesting bug over there (passing ComparableX509 to pyOpenSSL functions seems to work in later versions of the latter). We should audit the codebase and fix docstrings throughout. We probably also want `test_util.load_cert` and `test_util.load_comparable_cert`.

Also, how do you plan to test for regressions due to problems with `pyOpenSSL==0.13`. Current tests install latest version which would not catch all such errors. Maybe tests should be fixed to oldest version (of any package)?